vcs.go

```diff
@@ -110,59 +115,12 @@ func (g Git) OpenFile(revision, path string) (io.ReadCloser, error) {
 	return ioutil.NopCloser(bytes.NewReader(contents)), nil
 }
 
-func (g Git) DefaultRevision() (string, string) {
+// DefaultRevision returns the default revisions if none specified
+func (g *Git) DefaultRevision() (string, string) {
 	// Check if there's unstaged changes, if so, return dot
 	contents, _ := exec.Command("git", "--git-dir", g.dir, "ls-files", "-m").Output()
 	if len(contents) > 0 {
 		return "HEAD", "."
 	}
 	return "HEAD~1", "HEAD"
 }
-
-type fileInfo struct {
```

Comment:
fileInfo was still needed in git `ReadDir`, I don't think it can be moved to to the test file.